src/lib/mina_generators/zkapp_command_generators.mli

```diff
@@ -66,7 +66,8 @@ val gen_zkapp_command_from :
   -> ?failure:failure
   -> ?max_account_updates:int
   -> ?max_token_updates:int
-  -> ?map_account_update:(Account_update.t -> Account_update.t)
+  -> ?map_account_update:
```

Comment:
Wondering why you want to change this to `Stable.Latest.t`? Is there a convention?